tests/transforms/test_draw_mpl.py

```diff
@@ -68,34 +68,56 @@ def test_standard_use():
     plt.close()
 
 
-def test_fontsize():
-    """Test fontsize set by keyword argument."""
+class TestKwargs:
+    """Test various keywords arguments that can be passed to modify graphic."""
 
-    _, ax = qml.draw_mpl(circuit1, fontsize=20)(1.234, 1.234)
-    for t in ax.texts:
-        assert t.get_fontsize() == 20
+    def test_fontsize(self):
+        """Test fontsize set by keyword argument."""
 
+        _, ax = qml.draw_mpl(circuit1, fontsize=20)(1.234, 1.234)
+        for t in ax.texts:
+            assert t.get_fontsize() == 20
 
-def test_decimals():
-    """Test decimals changes operation labelling"""
+    def test_decimals(self):
+        """Test decimals changes operation labelling"""
 
```

Comment:
From this test I understand that if I have `RX(2)`, in the final drawing I should see `RX(2)`, however, when I run it I only see `RX`.

Reply:
(Just a question, I didn't want to modify the code.)

Reply:
![image](https://user-images.githubusercontent.com/6364575/144601026-f0a754e5-f13c-4f67-b605-0652830e5131.png)

f you want `RX(2)`, you should use the `decimals=0` keyword.  `decimals=None` is the default and doesn't include any parameters.

Reply:
From the documentation:
```
        decimals (int): How many decimal points to include when formatting operation parameters.
            Default ``None`` will omit parameters from operation labels.
```

Reply:
Ah ok, I was waiting by default to see it, but you're right 👍🙂